balena/models/device_os.py

```diff
@@ -107,6 +119,49 @@ def download(self, raw=None, **data):
         else:
             return response
 
+    def download_unconfigured_image(self, device_type, version='latest', raw=False):
+        """
+        Download an unconfigured OS image. This function only works if you log in using credentials or Auth Token.
```

Comment:
I just wanted to clarify that the API allows listing the versions & downloading unconfigured images w/o requiring authentication.
See: https://github.com/balena-io/balena-api/blob/8c3151df679357a5783b1142cdb0901f778ef8a4/src/routes/index.ts#L208-L209

So technically we could allow users do so w/o requiring authentication.

Reply:
Updated